packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -974,6 +981,39 @@ class DescopeWc extends BaseDescopeWc {
     }
   }
 
+  #updateExternalInputs() {
+    clearPreviousExternalInputs();
+
+    const eles = this.rootElement.querySelectorAll('[external-input="true"]');
+    eles.forEach((ele) => this.#handleExternalInputs(ele));
+  }
+
+  #handleExternalInputs(ele: Element) {
+    if (!ele) {
+      return;
+    }
+
+    if (ele.getAttribute('external-input') !== 'true') {
```

Comment:
Is this needed? the selector is for `[external-input="true"]`

Reply:
Removed.